_drafts/SC4-1-2-button-has-name.md

```diff
@@ -0,0 +1,85 @@
+---
+name: Buttons have an accessible name
+description: |
+  Each button element has an accessible name
+
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects: # Remove what is not applicable
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+- Stein Erik Skotkjerra
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any element with the role of `button`, except for `input` elements of `type="image"` that are [visible on the page][] or [exposed to assistive technologies][].
+
+### Expectation
+
+Each target element has an [accessible name][] that is [non-empty][].
+
+## Assumptions
+
+- The rule assumes that all buttons are user interface components as defined by WCAG 2. When the button role is used incorrectly, the assumption may not be true.
```

Comment:
Do we need the second sentence? The assumption is already defined in the first sentence.

Reply:
You are right. First sentence does the trick

Reply:
Is it necessary to clarify in th Applicability section that any element with the role of `button` also include HTML elements with implied role of button? example 
``input `type=submit`` has role=button.